node/src/utils.rs

```diff
@@ -46,22 +48,23 @@ pub fn insert_controller_account_keys_into_keystore(
 
 /// Inserts keys of specified type into the keystore.
 fn insert_account_keys_into_keystore<TPublic: Public>(
-	config: &Configuration,
+	_config: &Configuration,
 	key_type: KeyTypeId,
 	key_store: Option<KeystorePtr>,
 	key_name: &str,
 ) {
-	let seed = &config.network.node_name[..];
+	let seed: String =
+		rand::thread_rng().sample_iter(&Alphanumeric).take(30).map(char::from).collect();
 
```

Comment:
I would say generate 32 random bytes and then just use `hex::encode(seed)` instead of generating random strings?